inventory/ci

```diff
@@ -16,6 +16,9 @@ nodepool.bonnyci-internal.portbleu.com
 [log]
 logs.bonnyci-internal.portbleu.com
 
+[monitoring]
```

Comment:
Don't forget about the other inventories, we should add entries to 'inventory/allinone', 'inventory/nodepool.py', and 'inventory/vagrant' too.